README.md

```diff
@@ -1 +1,184 @@
+INSERT INTO cats_owners (cat_id, owner_id) VALUES (1, 2);
+```
+
+### Code Along I: `ORDER BY()`
+
+#### Syntax
+
+```slq
+SELECT column_name, column_name
+FROM table_name
+ORDER BY column_name ASC|DESC, column_name ASC|DESC;
+```
+
+`ORDER BY()` will automatically sort the returned values in ascending order. Use the `DESC` keyword, as above, to sort in descending order. 
+
+#### Exercise
+
+Imagine you're working for an important investment firm in Manhattan. The investors are interested in investing in a lucrative and popular cat. They need your help to decide which cat that will be. They want a list of famous and wealthy cats. We can do that with a basic SELECT statement:
+
+```sql
+SELECT * FROM cats WHERE net_worth > 0;
+```
+
+This will return:
+
+```
+name             age         breed          net_worth 
+---------------  ----------  -------------  ----------
+Maru             3           Scottish Fold  1000000   
+Hana             1           Tabby          21800     
+Grumpy Cat       4           Persian        181600    
+Lil' Bub         2           Tortoiseshell  2000000   
+```   
+
+Our investors are busy people though. They don't have time to manually sort through this list of cats for the best candidate. They want you to return the list to them with the cats sorted by net worth, from greatest to least.  
+
+We can do so with the following lines:
+
+```sql
+SELECT * FROM cats ORDER BY(net_worth) DESC;
+```
+
+This will return:
+
+```
+name             age         breed          net_worth 
+---------------  ----------  -------------  ----------
+Lil' Bub         2           Tortoiseshell  2000000   
+Maru             3           Scottish Fold  1000000   
+Grumpy Cat       4           Persian        181600    
+Hana             1           Tabby          21800     
+Nona             4           Tortoiseshell         
+```
+
+#### The `LIMIT` Keyword
+
+Turns out our investors are very impatient. They don't want to review the list themselves, they just want you to return to them the wealthiest cat. We can accomplish this by using the `LIMIT` keyword with the above query:
+
+```sql
+SELECT * FROM cats ORDER BY(net_worth) DESC LIMIT 1;
+name             age         breed          net_worth 
+---------------  ----------  -------------  ----------
+Lil' Bub         2           Tortoiseshell  2000000   
+```
+
+The LIMIT keyword specifies how many of records that result from the query you'd like to actually return. 
+
+### Code Along VII: `GROUP BY()`
+
+The `GROUP BY()` keyword is very similar to `ORDER BY()`. The only difference is that `ORDER BY()` sorts the resulting data set of basic queries while `GROUP BY()` sorts the result sets of aggregate functions. 
+
+#### Syntax
+
+```sql
+SELECT column_name, aggregate_function(column_name)
+FROM table_name
+WHERE column_name operator value
+GROUP BY column_name;
+```
+
+#### Exercise
+
+Let's calculate the sum of the net worth of all of the cats that belong to our second owner:
+
+```sql
```

Comment:
Did Sum without really introducing it. Also This query should be line by line instead of one big one, so hard to read